src/web/components/App.tsx

```diff
@@ -327,7 +337,50 @@ export const App = ({ CAPI, NAV, ophanRecord }: Props) => {
 				pubData,
 			});
 		}
-	}, [countryCode, CAPI.config.switches.consentManagement, pageViewId]);
+	}, [
+		countryCode,
+		CAPI.config.switches.consentManagement,
+		pageViewId,
+		browserId,
```

Comment:
Adding `browserId` here (where it is state being set in another `useEffect`) may add a delay and/or cause this code to run more often

Reply:
I think then maybe `useOnce` is the right concept, here? We need to have a `countryCode`, the switches, and a `browserId` + `pageViewId` pair, but these should never change during the page lifecycle.

Reply:
Also, `cmp.init` can only run once. If it’s `init`-ed, it won’t init again.